### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by jgleissner**
This PR mostly adds documentation on the change log generator and the OBS source service, plus some smaller fixes.

### CODE REVIEW

File: doc/source/changelog_generator.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
+.. _changelog_generator:
+
+Generating Change Logs
+======================
+
+`Keg` comes with a separate tool that can be used to produce a change log
+for a generated image description from the git commit history of the used
+`keg recipes` tree(s). This obviously requires these `keg recipes` to be
+stored in a git repositories.
```
Comment (aosthof) [inline doc/source/changelog_generator.rst:9]: Superfluous 'a' if repos are plural.

File: doc/source/changelog_generator.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+For the change log generator to work, the image description for which the
+change log(s) need to have been generated with source info tracking, which
+can be enabled using the `-s` command line switch.
```
Comment (aosthof) [inline doc/source/changelog_generator.rst:13]: This sentence sounds bumpy. I read it 3 times and still don't know 100% what it tries to tell me.

File: doc/source/changelog_generator.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+Source Info Tracking
+--------------------
+
+With source info tracking enabled, `keg` will write one or more source info
+files in addtion to the image description in the output directory. In case
```
Comment (aosthof) [inline doc/source/changelog_generator.rst:19]: Missing 'i' in addition.

File: doc/source/obs_source_service.rst [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+           <param name="git-recipes">https://github.com/SUSE-Enceladus/keg-recipes.git</param>
+           <param name="git-branch">released</param>
+           <param name="image-source">cross-cloud/sles/byos/15-sp3</param>
+       </service>
+   </services>
+
+In this example, the public `keg-recipes` for SUSE Linux Enterprise images
+hosted on github is used as source, working off the `released` branch,  and the
+selected image source is `cross-cloud/sles/byos/15-sp3`. Running the source
+service will produce a description for  SUSE Linux Enterprise Server 15 SP3
```
Comment (aosthof) [inline doc/source/obs_source_service.rst:32]: 2 blank spaces between 'for' and 'SUSE'.

File: doc/source/obs_source_service.rst [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+   <services>
+       <service name="compose_kiwi_description">
+           <param name="git-recipes">https://github.com/SUSE-Enceladus/keg-recipes.git</param>
+           <param name="git-branch">released</param>
+           <param name="image-source">cross-cloud/sles/byos/15-sp3</param>
+       </service>
+   </services>
+
+In this example, the public `keg-recipes` for SUSE Linux Enterprise images
+hosted on github is used as source, working off the `released` branch,  and the
```
Comment (aosthof) [inline doc/source/obs_source_service.rst:30]: Not 100% sure, but afaik it's "working off of the..."

**Review by aosthof — CHANGES_REQUESTED** [review]
Some typos to be corrected, else LGTM.

File: doc/source/changelog_generator.rst (hunk 1 above)
Comment (jgleissner) [reply doc/source/changelog_generator.rst]: Fixed.

File: doc/source/changelog_generator.rst (hunk 2 above)
Comment (jgleissner) [reply doc/source/changelog_generator.rst]: I've reworded it. Let me know if it's still unclear.

File: doc/source/changelog_generator.rst (hunk 3 above)
Comment (jgleissner) [reply doc/source/changelog_generator.rst]: Fixed.

File: doc/source/obs_source_service.rst (hunk 4 above)
Comment (jgleissner) [reply doc/source/obs_source_service.rst]: Fixed.

File: doc/source/obs_source_service.rst (hunk 5 above)
Comment (jgleissner) [reply doc/source/obs_source_service.rst]: It sounded a bit strange anyway. I've reworded it.

**Review by rjschwei — APPROVED** [review]

**Pull request merged**